src/systems/ec/implementations/react/templates/pages/event-detail/examples/Default.jsx

```diff
@@ -0,0 +1,815 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { Fragment } from 'react';
+
+import Breadcrumb, { BreadcrumbItem } from '@ecl/ec-react-component-breadcrumb';
+import Button from '@ecl/ec-react-component-button';
+import Card from '@ecl/ec-react-component-card';
+import DateBlock from '@ecl/ec-react-component-date-block';
+import Footer from '@ecl/ec-react-component-footer';
+import Icon from '@ecl/ec-react-component-icon';
+import LanguageListOverlay from '@ecl/ec-react-component-language-list/src/LanguageListOverlay';
+import Link from '@ecl/ec-react-component-link';
+import MediaContainer from '@ecl/ec-react-component-media-container';
+import PageBanner from '@ecl/ec-react-component-page-banner';
+import SiteHeader from '@ecl/ec-react-component-site-header';
+import SocialMediaFollow from '@ecl/ec-react-component-social-media-follow';
+import {
+  UnorderedList,
+  UnorderedListItem,
+} from '@ecl/ec-react-component-unordered-list';
+
+import footerContent from '@ecl/ec-specs-footer/demo/data--corporate';
+import languageListContent from '@ecl/ec-specs-language-list/demo/data--overlay';
+import siteHeaderContent from '@ecl/ec-specs-site-header/demo/data--en';
+
+export default () => {
+  return (
+    <Fragment>
+      <SiteHeader {...siteHeaderContent} />
+      <LanguageListOverlay {...languageListContent} hidden="true" />
+      {/* :'( Custom page header to add button */}
+      <div className="ecl-page-header">
+        <div className="ecl-container">
+          <Breadcrumb className="ecl-page-header__breadcrumb">
+            <BreadcrumbItem label="Home" href="/example" />
+            <BreadcrumbItem label="Events" href="/example" />
+            <BreadcrumbItem label="Fair of European Innovators in Cultural heritage" />
+          </Breadcrumb>
+          <div className="ecl-page-header__meta-list">Conference</div>
+          <div className="ecl-u-d-flex ecl-u-flex-column ecl-u-flex-md-row">
+            <h1 className="ecl-page-header__title ecl-u-flex-grow-1">
+              Fair of European Innovators in Cultural heritage
+            </h1>
+            <div className="ecl-u-align-self-end ecl-u-mt-l ecl-u-mt-md-none">
+              <Button variant="call" label="Register" />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <main className="ecl-u-pv-2xl ecl-u-pv-md-3xl">
+        <div className="ecl-container">
+          <div className="ecl-row">
+            <div className="ecl-col-12 ecl-col-md-6 ecl-u-order-md-last">
+              <UnorderedList variant="no-bullet">
```

Comment:
I don't see the value of using a list here

Reply:
I don't see why we shouldn't use a list... That's a "serie" of several items related to the same topic (event information). So to me that's clearly a list